lib/components/src/escapeHatch.ts

```diff
@@ -0,0 +1,7 @@
+import React, { FunctionComponent } from 'react';
+
+export function escapeHatch(component: FunctionComponent): string {
+  const name = component.displayName || component.name || 'component';
+  const cleanName = `sbdocs sbdocs-${name.toLowerCase()}`;
```

Comment:
I dont think this is enought for kebab case.

You still need to add a "-" between words.

For examples "ResetWrapper" should translate to "reset-wrapper".

Here's a few gist: https://gist.github.com/thevangelist/8ff91bac947018c9f3bfaad6487fa149

There is also a few NPM packages:

https://www.npmjs.com/package/kebab-case
https://www.npmjs.com/package/dashify
https://www.npmjs.com/package/decamelize

`decamelize` could be a great idea since it's already a sub dependency.